spec/charts/count_chart/count_chart_spec.rb

```diff
@@ -176,23 +176,23 @@
     it 'creates the prepared_data for simple keys' do
       chart = Charts::CountChart.new([3, 2], colors: ['x', 'o'], columns: 2)
       expect(chart.prepared_data).to eq([
-                                          ['x', 'x'],
-                                          ['x', 'o'],
-                                          ['o']
-                                        ])
+        ['x', 'x'],
+        ['x', 'o'],
+        ['o']
+      ])
     end
     it 'creates the prepared_data for complex keys' do
       chart = Charts::CountChart.new([2, 2], colors: ['#FF0000', '#00FF00'], columns: 2)
       expect(chart.prepared_data).to eq([
-                                          ['#FF0000', '#FF0000'],
-                                          ['#00FF00', '#00FF00']
-                                        ])
+        ['#FF0000', '#FF0000'],
+        ['#00FF00', '#00FF00']
+      ])
     end
     it 'default colors get assigned when no colors are specified' do
       chart = Charts::CountChart.new([1, 1, 1])
       expect(chart.prepared_data).to eq([
-                                          ['#e41a1d', '#377eb9', '#4daf4b']
-                                        ])
+        ['#e41a1d', '#377eb9', '#4daf4b']
+      ])
```

Comment:
This one is short enough to be conveniently on the same line, I think.  Don't expand them out until necessary.